src/main/routes.ts

```diff
@@ -43,19 +42,16 @@ export class Routes {
     app.get(ACCESSIBILITY_STATEMENT_URL, errorHandler(new AccessibilityStatementGetController().get));
 
     for (const step of stepsWithContent) {
-      const stepDir = `${__dirname}/steps${step.url}`;
-      const customView = `${stepDir}/template.njk`;
-      const view = fs.existsSync(customView) ? customView : `${stepDir}/../common/template.njk`;
-      const controller = new GetController(view, step.generateContent);
+      const dir = `${__dirname}/steps${step.url}`;
+      const customView = `${dir}/template.njk`;
+      const view = fs.existsSync(customView) ? customView : `${dir}/../common/template.njk`;
+      const getController = fs.existsSync(`${dir}/get.ts`) ? require(`${dir}/get.ts`).default() : GetController;
 
-      app.get(step.url, errorHandler(controller.get));
+      app.get(step.url, errorHandler(new getController(view, step.generateContent).get));
 
       if (step.form) {
-        if (jurisdictionSteps.map(String).includes(step.url)) {
-          app.post(step.url, errorHandler(new JurisdictionPostController(new Form(step.form)).post));
-        } else {
-          app.post(step.url, errorHandler(new PostController(new Form(step.form)).post));
-        }
+        const postController = fs.existsSync(`${dir}/post.ts`) ? require(`${dir}/post.ts`).default() : PostController;
```

Comment:
Does this work by looking into the directory and seeing if there is a custom post? I'm just thinking that there's 5 pages the custom post controller needs to be called on, but it lives inside the jurisdiction/interstitial directory